package.json

```diff
@@ -14,7 +14,7 @@
     "lint-config": "eslint ./*.js",
     "lint-react": "eslint --ext .js,.jsx src",
     "lint-test": "eslint --ext .js,.jsx test",
-    "npm:preversion": "builder run lint && builder run build-static",
+    "npm:preversion": "builder run lint && --env='{"NODE_ENV":"production"}' builder run build-static",
```

Comment:
Oops, fixed.
 
What do you mean by your last question? The task will create a `build` that doesn’t get checked into github, right? So, what am I checking in?